### PR TITLE
Integration of SI and PVA

### PR DESCRIPTION
**Pull request by pmoharan**

### CODE REVIEW

File: src/bcs_oi_api/bcs_oi_api.py
Comment (mnuvvuru) [inline src/bcs_oi_api/bcs_oi_api.py:1]: pls remove localhost & uncomment server host

Comment (mnuvvuru) [inline src/bcs_oi_api/bcs_oi_api.py:1]: Can you check once table name mappings once?

File: src/bcs_oi_api/models/policy_variation_analysis/__init__.py
Comment (mnuvvuru) [inline src/bcs_oi_api/models/policy_variation_analysis/__init__.py:1]: PvapoliciesFilter should be in camel case.

File: src/bcs_oi_api/models/policy_variation_analysis/models.py
Comment (mnuvvuru) [inline src/bcs_oi_api/models/policy_variation_analysis/models.py:1]: PvapoliciesFilter should be in camel case.

File: src/bcs_oi_api/models/product_alerts/models.py
Comment (mnuvvuru) [inline src/bcs_oi_api/models/product_alerts/models.py:1]: >>>>>>> 9908aa1ddc09bdb96071b9a3d21fe234227dddc0
May I know why it got added?

File: src/bcs_oi_api/models/policy_variation_analysis/__init__.py
Comment (pmoharan) [reply src/bcs_oi_api/models/policy_variation_analysis/__init__.py:1]: done.

File: src/bcs_oi_api/models/policy_variation_analysis/models.py
Comment (pmoharan) [reply src/bcs_oi_api/models/policy_variation_analysis/models.py:1]: done.

File: src/bcs_oi_api/bcs_oi_api.py
Comment (pmoharan) [reply src/bcs_oi_api/bcs_oi_api.py:1]: done removed

Comment (pmoharan) [reply src/bcs_oi_api/bcs_oi_api.py:1]: okay checked.

File: src/bcs_oi_api/models/product_alerts/models.py
Comment (pmoharan) [reply src/bcs_oi_api/models/product_alerts/models.py:1]: removed

File: tests/product_alerts/test_product_alerts.py
Comment (mnuvvuru) [inline tests/product_alerts/test_product_alerts.py:1]: can you pls update replacement attribute name

Comment (pmoharan) [reply tests/product_alerts/test_product_alerts.py:1]: done

**Pull request merged**